README.md

```diff
@@ -91,8 +91,96 @@ scripts, which you can copy to your current working directory by running
 [the pyd docs](https://pyd.readthedocs.io/en/latest/dub.html) for more
 information.
 
+# .NET Support
+
+Autowrap can also generate C# bindings to D libraries. Generating bindings for C# is a two-step process. First we must generate a C# compatible C-interface for the D library, then we build and run a version of the library that emits the corresponding C# interface code.
+
+### What Works:
+
+* Primitive Types
+  * string
+  * wstring
+  * dstring
+  * byte
+  * ubyte
+  * short
+  * ushort
+  * int
+  * uint
+  * long
+  * ulong
+  * float
+  * double
+* Structs
+  * Fields
+  * Properties
+  * Methods
+* Classes
+  * Constructors
+  * Fields
+  * Properties
+  * Methods
+
+1-Dimensional Ranges are implemented but only lightly tested and may not work in all cases. Higher dimensional ranges are not supported.
+
+## Generating .NET Interfaces
+
+To generate the C-interface use the following example replacing "csharp" with the dub name of your library (ex: csharp builds to libcsharp&#46;so) and replace "csharp.library" with the name of the module you want to wrap.
 
-### Our sponsors
+```d
+import csharp.library;
```

Comment:
?

Reply:
It's still at the bottom, I added the .NET stuff above it.

Reply:
I meant: why is this being imported?